msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md

```diff
@@ -35,20 +35,24 @@ The Teams AI Library is available in JavaScript and C# languages allowing you to
 
 The following are some of the main features available through Teams AI library:
 
-* [Simple Teams-centric component scaffolding](#simple-teams-centric-component-scaffolding)
-* [Natural language modeling](#natural-language-modeling)
-* [Prompt engineering](#prompt-engineering)
-* [Conversational session history](#conversational-session-history)
-* [Localization](#localization)
-* [LLM modularity](#llm-modularity)
-* [Responsible AI](#responsible-ai)
-* [Predictive engine for mapping intents to actions](#predictive-engine-for-mapping-intents-to-actions)
-* [Action Planner](#action-planner)
-* [Prompt management](#prompt-management)
-* [Augmentation](#augmentation)
-* [Vector data sources](#vector-data-sources)
-* [Enhanced reasoning](#enhanced-reasoning)
-* [Feedback loop](#feedback-loop)
+- [Teams AI library](#teams-ai-library)
+  - [Why use Teams AI library?](#why-use-teams-ai-library)
+  - [Simple Teams-centric component scaffolding](#simple-teams-centric-component-scaffolding)
+  - [Natural language modeling](#natural-language-modeling)
+  - [Prompt engineering](#prompt-engineering)
+  - [Conversational session history](#conversational-session-history)
+  - [Localization](#localization)
+  - [LLM modularity](#llm-modularity)
+  - [Responsible AI](#responsible-ai)
+  - [Predictive engine for mapping intents to actions](#predictive-engine-for-mapping-intents-to-actions)
+  - [Action Planner](#action-planner)
+  - [Prompt management](#prompt-management)
+  - [Augmentation](#augmentation)
+    - [Vector data sources](#vector-data-sources)
+  - [Enhanced reasoning](#enhanced-reasoning)
+  - [Feedback loop](#feedback-loop)
+  - [Code samples](#code-samples)
+  - [Next step](#next-step)
 
```

Comment:
Hi @michaelmaillot,

Thank you for contributing to the docs. We accept your suggestion; however, we can't accept this change as we already have a heading Index at the beginning of the article.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/0978ddf3-a5dd-4d71-ab8c-0fc738bc7074)

We will remove all the heading links from this section, and we will merge PR tomorrow.